packages/botonic-react/tests/webchat.theme.test.js

```diff
@@ -0,0 +1,72 @@
+import { getProperty } from '../src/utils'
+import { CUSTOM_WEBCHAT_PROPERTIES } from '../src/constants'
+
+let theme = {
+  message: {
+    bot: {
+      image: 'DefaultLogoPathNested',
+      style: {
+        border: 'none',
+        color: 'blue',
+        borderRadius: '20px',
+      },
+    },
+  },
+  botMessageImage: 'DefaultLogoPath',
+}
+
+let anotherTheme = {
+  message: {
+    bot: {
+      image: null,
+      style: {
+        border: 'none',
+        color: 'blue',
+        borderRadius: '20px',
+      },
+    },
+  },
+  botMessageImage: null,
+}
+
+const getThemeProperty = (property, defaultValue = undefined) => {
```

Comment:
you should never test a copy of the real code. 
You can move getThemeProperty to utils.js with getProperty, or move both to a new properties.js
to inject the them, one way is having a higher order function:
```
function getThemeProperty(theme) {
   return (property, defaultValue = undefined) => {
       ...
   }
}
```

Reply:
now fixed and refactored, passing the test as expected 👍